pages/home.page.ts

```diff
@@ -0,0 +1,23 @@
+import { Locator, Page } from "@playwright/test";
+import { HeaderFragment } from './fragments/header.fragment';
+
+
+export class HomePage{
+    page: Page;
+    header: HeaderFragment;
+
+    banner: Locator;
+    
+
+
+    constructor(page: Page){
+        this.page = page;
+        this.header = new HeaderFragment(page);
+        this.banner = this.page.locator('.img-fluid');
+       
+    }
+
+     getCardByTitle(title: string): Locator {
+        return this.page.locator(`.card:has(h5:has-text("${title}"))`);
```

Comment:
Consider using [filter()](https://playwright.dev/docs/next/api/class-locator#locator-filter) instead of overcomplicated css selectors. This option is cleaner and easier to read than a long, nested CSS selector, which mean easier to maintain and debug in the future. 
```suggestion
        return this.page.locator('.card').filter({ has: this.page.getByText(title) });
```